### PR TITLE
TX payment query API for Dali #CU-8669j3pbp

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

File: hyperspace/parachain/src/chain.rs [hunk 1]
```diff
@@ -132,13 +137,24 @@ where
...
+		let dispatch_info =
+			TransactionPaymentApiClient::<sp_core::H256, RuntimeDispatchInfo>::query_info(
+				&*self.para_ws_client,
+				extrinsic.into(),
+				None,
+			)
+			.await
+			.map_err(|e| Error::from(format!("Rpc Error From Estimating weight {:?}", e)))?;
+
+		#[cfg(feature = "dali")]
```
Comment (blasrodri) [inline hyperspace/parachain/src/chain.rs:149]: it will be the same for Picasso and composable right?

Comment (blasrodri) [reply hyperspace/parachain/src/chain.rs:149]: if so, then we can just add it right now

**Review by blasrodri — APPROVED** [review]

File: hyperspace/parachain/src/chain.rs (hunk 1 above)
Comment (dzmitry-lahoda) [reply hyperspace/parachain/src/chain.rs:149]: same

Comment (dzmitry-lahoda) [reply hyperspace/parachain/src/chain.rs:149]: just used already existing feature.

**Comment by dzmitry-lahoda** [comment]
As I see tests involving my changes passed (anyway my changes should be no change to these tests, only to dali tests). So merging it to reuse in monorepo.

**Review by obsessed-cake — APPROVED** [review]

**Comment by blasrodri** [comment]
@dzmitry-lahoda be patient and wait until CI passes

**Comment by dzmitry-lahoda** [comment]
@blasrodri sure. sorry. will do next time.

**Pull request merged**